### PR TITLE
Separated out DB contracts

### PR DESCRIPTION
**Pull request by hkasera**
I have separated out DB contracts in a separate file `PowerUpContract.java` as we discussed in Issue #74 . Please check it out and let me know. 

Thanks a lot,
Harshita

### CODE REVIEW

File: PowerUp/app/src/main/java/powerup/systers/com/db/PowerUpContract.java [hunk 1]
```diff
@@ -0,0 +1,91 @@
+package powerup.systers.com.db;
+
+import android.provider.BaseColumns;
+
+/**
+ * Created by harshita.k on 19/01/16.
+ */
```
Comment (alma-castant) [inline PowerUp/app/src/main/java/powerup/systers/com/db/PowerUpContract.java]: Please remove this heading as we don't use them

File: PowerUp/app/src/main/java/powerup/systers/com/db/AbstractDbAdapter.java [hunk 2]
```diff
@@ -165,9 +191,9 @@ public void onCreate(SQLiteDatabase db) {
 
 		@Override
 		public void onUpgrade(SQLiteDatabase db, int arg1, int arg2) {
-			db.execSQL("DROP TABLE IF EXISTS Question");
-			db.execSQL("DROP TABLE IF EXISTS Answer");
-			db.execSQL("DROP TABLE IF EXISTS Scenario");
+			db.execSQL("DROP TABLE IF EXISTS "+PowerUpContract.QuestionEntry.TABLE_NAME);
```
Comment (alma-castant) [inline PowerUp/app/src/main/java/powerup/systers/com/db/AbstractDbAdapter.java]: Leave a space before and after `+`. if you run Code->Reformat Code on Android Studio, you will get the right formatting

File: PowerUp/app/src/main/java/powerup/systers/com/db/AbstractDbAdapter.java [hunk 3]
```diff
@@ -165,9 +191,9 @@ public void onCreate(SQLiteDatabase db) {
...
+			db.execSQL("DROP TABLE IF EXISTS "+PowerUpContract.AnswerEntry.TABLE_NAME);
```
Comment (alma-castant) [inline PowerUp/app/src/main/java/powerup/systers/com/db/AbstractDbAdapter.java]: Same

File: PowerUp/app/src/main/java/powerup/systers/com/db/AbstractDbAdapter.java [hunk 4]
```diff
@@ -165,9 +191,9 @@ public void onCreate(SQLiteDatabase db) {
...
+			db.execSQL("DROP TABLE IF EXISTS "+PowerUpContract.ScenarioEntry.TABLE_NAME);
```
Comment (alma-castant) [inline PowerUp/app/src/main/java/powerup/systers/com/db/AbstractDbAdapter.java]: Same

**Comment by alma-castant** [comment]
The code is good is works well. I left a comment on some details that need to be fixed. Please amend your commit using `git commit --amend` so your pull request is one commit only. Leave a comment here after you update the PR so I get notified ;)

**Comment by hkasera** [comment]
Thank you for the neat review @alma-castant. I have made the changes and updated the Pull Request.

**Pull request merged**